core/store/src/flat/types.rs

```diff
@@ -81,6 +83,12 @@ impl Into<i64> for &FlatStorageStatus {
                 FlatStorageCreationStatus::FetchingState(_) => 11,
                 FlatStorageCreationStatus::CatchingUp(_) => 12,
             },
+            // 20..30 is reserved for resharding statuses.
+            FlatStorageStatus::Resharding(resharding_status) => match resharding_status {
+                FlatStorageReshardingStatus::SplittingParent => 20,
```

Comment:
Unsure about what's the difference between SplittingParent and CreatingChild?

Also, to be considered, we may need to provide some input in terms of which children we are tracking.

Reply:
They are two faces of the same medal, different names for clarity. The reasoning is that if we have to assign a status to each shard, the parent would be splitting and the child creating